src/main/java/trackitnus/logic/commands/module/EditModuleCommand.java

```diff
@@ -168,12 +159,11 @@ public void setName(Name name) {
             this.name = name;
         }
 
-        public Optional<String> getDesc() {
-            return Optional.ofNullable(desc);
-        }
-
-        public void setDesc(String desc) {
-            this.desc = desc;
+        @Override
+        public String toString() {
+            return "EditModuleDescriptor{" +
+                "name=" + name +
+                '}';
```

Comment:
Why do we need this? And Maybe change to `Name: ` is this for testing purposes?

Reply:
for testing only